node_modules/gh-core/lib/db.js

```diff
@@ -468,8 +468,32 @@ var _setUpModel = function(sequelize) {
     Serie.belongsTo(Group);
     Group.hasMany(Serie);
 
-    OrgUnit.hasMany(Serie);
-    Serie.hasMany(OrgUnit);
+    /**
+     * The join table between the OrgUnit and Serie tables. This table is defined explicitly,
+     * as Sequelize doesn't intuitively support three way joins.
+     *
+     * This table will hold 3 values:
+     *  - The organisational unit that contains a serie
+     *  - The serie that is in the organisational unit
+     *  - An optional organisational unit where the serie was "borrowed" from
+     *
+     * This is to allow for the timetable use-case where series can be borrowed from each other
```

Comment:
Lets take the following scenario: Serie X was created originally under Organisational Unit A. Serie X was then borrowed under Organisational Unit B. An administrators of Organisational Unit C then goes to the Organisational Unit B admin page and indicates that she wants to borrow Serie X. Should timetable then display that the serie was borrowed from Organisational Unit B or should I continue to point to Organisational Unit A.

If the latter, can we not determine which module a serie is borrowed from by taking all of the modules that use the serie and taking the one to which it was added first?